python/tvm/relay/op/strategy/cuda.py

```diff
@@ -698,6 +698,26 @@ def conv1d_transpose_strategy_cuda(attrs, inputs, out_type, target):
     return strategy
 
 
+@matmul_strategy.register(["cuda", "gpu"])
+def matmul_strategy_cuda(attrs, inputs, out_type, target):
```

Comment:
it's highly possible that this function generates no strategy. We need a checker to throw an error as well as the workaround for this situation.